components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMRoleManagerV2.java

```diff
@@ -98,6 +101,7 @@ public class SCIMRoleManagerV2 implements RoleV2Manager {
     private static final String ROLE_NAME_FILTER_ATTRIBUTE = "name";
     private static final String ROLE_AUDIENCE_TYPE_FILTER_ATTRIBUTE = "audience";
     private static final String ROLE_AUDIENCE_ID_FILTER_ATTRIBUTE = "audienceId";
+    private static final String APPLICATION_ROLE_AUDIENCE = "APPLICATION";
```

Comment:
Shall we use https://github.com/wso2/carbon-identity-framework/blob/eb81cb03e8f05f0654fdbd7778d4249daf2aeb50/components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/RoleConstants.java#L66 constant here

Reply:
Updated